src/app

```diff
@@ -28,7 +28,7 @@ cleanup() {
 trap cleanup 1 2 3 6 15
 
 CONFIG_FILE="/etc/mender/mender-app.conf"
-PERSISTENT_STORE="/data/mender-app"
+PERSISTENT_STORE="/var/lib/mender/app-modules"
```

Comment:
I wonder why the tests passed.

Reply:
seriously, I am against assuming this directory just exists. we will need to remember about it in many places (debs also, apart from yocto and tests -- which we will do). cant we try to make the directory in case it does not exist and we can make it?

Reply:
> cant we try to make the directory in case it does not exist and we can make it?

Sure! Let's do it :+1: